[Workers] Clarify log size limits (#10033)
cloudflare/cloudflare-docs: content/workers/platform/limits.md

## Image Resizing with Workers 

When using Image Resizing with Workers, refer to [Image Resizing documentation](/images/image-resizing/format-limitations/#limits-per-format) for more information on the applied limits.

---

> ## Log size

Author: @deadlypants1973 — feel free to move around if you see a better home for this content on this page.

Reviewer: This whole page needs a bit of an audit (some dupes and formating stuff) so I will take care of it then :)

> - You can log (`console.log()`) a maximum of 128 KB of data to the console for a single request. After you exceed this limit, further log statements will not be recorded or appear when tailing the logs of your Worker.

Author: ```suggestion
- You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, reqeust metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](workers/observability/tail-workers/).
```

> - You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, reqeust metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](workers/observability/tail-workers/).

Reviewer 2: ```suggestion
- You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, request metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](workers/observability/tail-workers/).
```

> - You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, request metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](workers/observability/tail-workers/).

Reviewer 3: ```suggestion
- You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, request metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](/workers/observability/tail-workers/).
```

> - You can log (`console.log()`) a maximum of 128KB of data to the console for a single request. After you exceed this limit, further log statements will not be recorded or appear when tailing the logs of your Worker.
> - Refer to the [Workers Trace Event Logpush docs](/workers/observability/logpush/#limits) for information on the maximum size of fields sent to logpush destinations.

Reviewer: ```suggestion
- You can log (`console.log()`) a maximum of 128 KB of data to the console for a single request. After you exceed this limit, further log statements will not be recorded or appear when tailing the logs of your Worker.
- Refer to the [Workers Trace Event Logpush documentation](/workers/observability/logpush/#limits) for information on the maximum size of fields sent to logpush destinations.
```
